books/free-programming-books-it.md

```diff
@@ -63,9 +62,10 @@
 * [Programmazione di Sistema in Linguaggio C - Esempi ed esercizi](https://www.disi.unige.it/person/DelzannoG/BIOMED/Programmazione-C/dispense_avanzate_C.pdf) - V. Gervasi, S. Pelagatti, S. Ruggieri, F. Scozzari, A. Sperduti (PDF)
 
 
-#### Sistemi di controllo versione
+#### <a id="version-control-systems"></a><a id="git"></a><a id="svn"></a><a id="subversion"></a>Sistemi di controllo versione
```

Comment:
Why add these links?

Reply:
Is more or less like happens in `-subjects.md`. It lefts splitting but here an H5 is not allowed so I think that adding a crosslink anchor is enought now that headings with html works in the parser EbookFoundation/free-programming-books-parser#2

Reply:
But those id aren't used anywhere

Reply:
Removed then. 570289b44b2cfff005b7a77ed5e31441015a1234

Reply:
Removed then. 570289b44b2cfff005b7a77ed5e31441015a1234